src/common/Layers.gd

```diff
@@ -8,4 +8,5 @@ enum Physics2D
 	ENEMY = 0b100,
 	INTERACT = 0b1000,
 	HEAT = 0b1_0000,
+	HOLE = 0b1_00000
 };
```

Comment:
the point of the _ is to group bits by the quads, so 0b110011001100 becomes 0b1100_1100_1100, making it much more readable.